mirdata/datasets/compmusic_jingju_acappella.py

```diff
@@ -0,0 +1,399 @@
+        syllable_path (str): local path where the syllable annotation is stored
+
+    Properties:
+        audio (tuple): track audio
+        work (str): string referring to the work where the trakc belongs
+        details (float): string referring to additional details about the track
+
+    Cached Properties:
+        phoneme (EventData): phoneme annotation
+        phrase_char (EventData): lyric phrase annotation in chinese
+        phrase (EventData): lyric phrase annotation in western characters
+        syllable (EventData): syllable annotation
+
+    """
+
+    def __init__(
+        self,
+        track_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            track_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        self.audio_path = self.get_path("audio")
+
+        self.phoneme_path = self.get_path("phoneme")
+        self.phrase_char_path = self.get_path("phrase_char")
+        self.phrase_path = self.get_path("phrase")
+        self.syllable_path = self.get_path("syllable")
+
+    @core.cached_property
+    def phoneme(self):
+        return load_phonemes(self.phoneme_path)
+
+    @core.cached_property
+    def phrase(self):
+        return load_phrases(self.phrase_path)
+
+    @core.cached_property
+    def phrase_char(self):
+        return load_phrases_char(self.phrase_char_path)
+
+    @core.cached_property
+    def syllable(self):
+        return load_syllable(self.syllable_path)
+
+    @property
+    def work(self):
+        return self._track_metadata.get("work")
+
+    @property
+    def details(self):
+        return self._track_metadata.get("details")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The track's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path,
+            lyrics_data=[(self.phrase, "phrases"), (self.phrase_char, "phrases_char")],
+            event_data=[(self.phoneme, "phoneme"), (self.syllable, "syllable")],
+            metadata={
+                "work": self.work,
+                "details": self.details,
+            },
+        )
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
+    """Load Jingju A Cappella Singing audio file.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to audio file
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+    """
+    return librosa.load(fhandle, sr=44100, mono=True)
```

Comment:
I am not sure, but do you get ssample rate as output if you pass the sr=44100 as input to librosa.load?

Reply:
Yes you do

Reply:
Indeed! Thanks @magdalenafuentes for the answer. The documentation states that all the tracks are sampled at 44.1kHz, that is why I decided to pass it as a parameter here.